safe_transaction_service/history/indexers/erc20_events_indexer.py

```diff
@@ -173,3 +182,40 @@ def process_elements(
             return range(
                 result_erc20 + result_erc721
             )  # TODO Hack to prevent returning `TokenTransfer` and using too much RAM
+
+    def get_almost_updated_addresses(
+        self, current_block_number: int
+    ) -> QuerySet[MonitoredAddress]:
+        """
+
+        :param current_block_number:
+        :return: Monitored addresses to be processed
+        """
+
+        logger.debug("%s: Retrieving monitored addresses", self.__class__.__name__)
+
+        addresses = self.database_queryset.all()
```

Comment:
The database_queryset function is :
```
    def database_queryset(self):
        return SafeContract.objects.all()
```
then here database_queryset.all() makes sense?

Reply:
Yes, as `all()` forces reevaluating